database/Models/index.js

```diff
@@ -7,22 +7,19 @@ const sequelize = new Sequelize({
   database: 'sdc',
   password: 'test',
   port: 5432,
-  host: 'localhost'
+  host: 'localhost',
+  logging: false
 });
 
 const models = [
```

Comment:
Could you run through lines 14-21? I'm a bit lost.

Reply:
Pretty much each model file returns a function, which returns the instance of the sequelize model.  Each one is stored (Only two, but if project grew this seemed like a scalable way to structure it) in this `models` array.  and then below and looping over that array of functions, and invoking each function with the main sequelize object.  Happy to explain on stand up if unclear.. I didn't come up with this method, but thought it was neat.. can be found here: https://github.com/sequelize/express-example